### ISSUE TITLE
Pre-work Checklist: Developer: Aswut Maxcy

### ISSUE THREAD

**Issue by aswutmaxcy** [hackforla/website#6424]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Self Assign this issue (gear in right side panel)
- [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
- [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (two per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by HackforLABot** [comment]
The Hack for LA Bot has closed this issue due to member inactivity.

**Comment by t-will-gillis** [comment]
Hello @aswutmaxcy we added you back on the team on [Oct 14th per your request.](https://github.com/hackforla/website/issues/6823#issuecomment-2411823123) Since then, it does not appear that you have had any activity. If you would like to remain on the Website team, you will need to provide weekly updates on this issue and on any issues you are assigned to.

If we do not hear from you in the next few days, we will assume you are not ready to come back to the team and you will be removed again.

**Comment by aswutmaxcy** [comment]
Rgr, sorry for the delay, I'll start working on issues tomorrow and keep this issue and any future ones I open updated.

**Comment by HackforLABot** [comment]
Hello @aswutmaxcy, we appreciate you taking on issue #7378, however it looks like you've already worked on enough issues of this complexity. Try checking out some of the issues of the next complexity from the Prioritized Backlog :)

We are going to unassign you from this issue so you can take on another issue.

Hfla appreciates you! :)

P.S. There is one exception to this rule/automation, and that is if you were away for a long time, and need to do the issue ladder again. If that is the case, please post the following note on the issue and on your Skills Issue (Pre-work Checklist). A Merge team member will reassign you to this issue, and will help you get assigned to subsequent issues up to medium size.

```
I am returning after a significant time away, and need to do the issue ladder again. Please assign me back to this issue.
```

**Comment by HackforLABot** [comment]
Hello @aswutmaxcy, we appreciate you taking on issue #7642, however it looks like you've already worked on enough issues of this complexity. Try checking out some of the issues of the next complexity from the Prioritized Backlog :)

We are going to unassign you from this issue so you can take on another issue.

Hfla appreciates you! :)

P.S. There is one exception to this rule/automation, and that is if you were away for a long time, and need to do the issue ladder again. If that is the case, please post the following note on the issue and on your Skills Issue (Pre-work Checklist). A Merge team member will reassign you to this issue, and will help you get assigned to subsequent issues up to medium size.

```
I am returning after a significant time away, and need to do the issue ladder again. Please assign me back to this issue.
```

**Comment by HackforLABot** [comment]
@aswutmaxcy

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, November 25, 2024 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@aswutmaxcy

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, December 30, 2024 at 11:04 PM PST.</sub>

**Comment by aswutmaxcy** [comment]
> @aswutmaxcy
> 
> Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
> 
>     1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
> 
>     2. Blockers: "Difficulties or errors encountered."
> 
>     3. Availability: "How much time will you have this week to work on this issue?"
> 
>     4. ETA: "When do you expect this issue to be completed?"
> 
>     5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."
> 
> 
> If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.
> 
> You are receiving this comment because your last comment was before Monday, December 30, 2024 at 11:04 PM PST.

    Progress: Starting new issue
    Blockers: None
    Availability: PST 9-5 m-f
    ETA: Unsure
    Pictures (optional): "Add any pictures of the visual changes made to the site so far."

**Comment by HackforLABot** [comment]
## Activity Log: aswutmaxcy
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- aswutmaxcy opened: https://github.com/hackforla/website/issues/6422 at 03/04/2024, 07:45 PM PST
- aswutmaxcy opened: https://github.com/hackforla/website/issues/6424 at 03/04/2024, 07:51 PM PST
- aswutmaxcy assigned: https://github.com/hackforla/website/issues/6424 at 03/04/2024, 07:52 PM PST
- aswutmaxcy closed as completed: https://github.com/hackforla/website/issues/6422#event-12008169997 at 03/04/2024, 08:30 PM PST
- aswutmaxcy assigned: https://github.com/hackforla/website/issues/6154 at 03/07/2024, 09:29 AM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/6154#issuecomment-1984142049 at 03/07/2024, 10:07 AM PST
- aswutmaxcy opened: https://github.com/hackforla/website/pull/6437 at 03/07/2024, 10:17 AM PST
- aswutmaxcy closed: https://github.com/hackforla/website/pull/6437#event-12046225196 at 03/07/2024, 10:18 AM PST
- aswutmaxcy reopened: https://github.com/hackforla/website/pull/6437#event-12046225196 at 03/07/2024, 10:19 AM PST
- aswutmaxcy assigned: https://github.com/hackforla/website/issues/6180 at 03/07/2024, 10:27 AM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/6180#issuecomment-1984177408 at 03/07/2024, 10:29 AM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/6180#issuecomment-1984446750 at 03/07/2024, 12:58 PM PST
- aswutmaxcy opened: https://github.com/hackforla/website/pull/6438 at 03/07/2024, 01:20 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/5541#issuecomment-1984535134 at 03/07/2024, 01:30 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/6180#issuecomment-1984535755 at 03/07/2024, 01:31 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/5541#issuecomment-1984537510 at 03/07/2024, 01:32 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/6180#issuecomment-1984537616 at 03/07/2024, 01:32 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/pull/6438#issuecomment-1985003358 at 03/07/2024, 08:05 PM PST
- aswutmaxcy unassigned: https://github.com/hackforla/website/issues/6154 at 03/07/2024, 11:32 PM PST
- aswutmaxcy PR was closed: https://github.com/hackforla/website/pull/6437#event-12052223239 at 03/07/2024, 11:37 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/pull/6438#issuecomment-1989196748 at 03/11/2024, 11:47 AM PDT
- aswutmaxcy PR was merged: https://github.com/hackforla/website/pull/6438#event-12079422538 at 03/11/2024, 01:06 PM PDT
- aswutmaxcy issue closed by PR 6438: https://github.com/hackforla/website/issues/6180#event-12079422756 at 03/11/2024, 01:06 PM PDT
- aswutmaxcy assigned: https://github.com/hackforla/website/issues/6169 at 03/13/2024, 10:10 AM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/6169#issuecomment-1995035013 at 03/13/2024, 10:11 AM PDT
- aswutmaxcy opened: https://github.com/hackforla/website/pull/6458 at 03/13/2024, 11:25 AM PDT
- aswutmaxcy closed: https://github.com/hackforla/website/pull/6458#event-12131240486 at 03/15/2024, 05:52 AM PDT
- aswutmaxcy opened: https://github.com/hackforla/website/pull/6465 at 03/15/2024, 06:28 AM PDT
- aswutmaxcy PR was merged: https://github.com/hackforla/website/pull/6465#event-12174881975 at 03/19/2024, 12:53 PM PDT
- aswutmaxcy issue closed by PR 6465: https://github.com/hackforla/website/issues/6169#event-12174882189 at 03/19/2024, 12:53 PM PDT
- aswutmaxcy assigned: https://github.com/hackforla/website/issues/6450 at 03/20/2024, 07:35 AM PDT
- aswutmaxcy unassigned: https://github.com/hackforla/website/issues/6450 at 03/20/2024, 07:36 AM PDT
- aswutmaxcy assigned: https://github.com/hackforla/website/issues/6487 at 03/21/2024, 06:01 AM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/6487#issuecomment-2012242682 at 03/21/2024, 06:03 AM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/6487#issuecomment-2014045472 at 03/21/2024, 04:31 PM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/6487#issuecomment-2015763567 at 03/22/2024, 12:22 PM PDT
- aswutmaxcy issue closed as completed: https://github.com/hackforla/website/issues/6487#event-12277794713 at 03/28/2024, 01:29 AM PDT
- aswutmaxcy issue closed as completed: https://github.com/hackforla/website/issues/6424#event-13008918393 at 06/01/2024, 04:07 AM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/6424#issuecomment-2453845048 at 11/03/2024, 09:16 PM PST
- aswutmaxcy assigned: https://github.com/hackforla/website/issues/7378 at 11/07/2024, 01:20 PM PST
- aswutmaxcy unassigned: https://github.com/hackforla/website/issues/7378 at 11/07/2024, 01:20 PM PST
- aswutmaxcy assigned: https://github.com/hackforla/website/issues/7642 at 11/07/2024, 01:30 PM PST
- aswutmaxcy unassigned: https://github.com/hackforla/website/issues/7642 at 11/07/2024, 01:30 PM PST
- aswutmaxcy assigned: https://github.com/hackforla/website/issues/7601 at 11/07/2024, 01:53 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/7601#issuecomment-2463306514 at 11/07/2024, 01:55 PM PST
- aswutmaxcy opened: https://github.com/hackforla/website/pull/7713 at 11/09/2024, 03:57 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/pull/7713#issuecomment-2469409294 at 11/11/2024, 05:30 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/pull/7713#issuecomment-2477356230 at 11/14/2024, 12:32 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/pull/7713#issuecomment-2477509753 at 11/14/2024, 02:12 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/pull/7713#issuecomment-2477958253 at 11/14/2024, 08:52 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/pull/7713#issuecomment-2480315223 at 11/15/2024, 06:04 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/pull/7713#issuecomment-2480316859 at 11/15/2024, 06:05 PM PST
- aswutmaxcy PR was merged: https://github.com/hackforla/website/pull/7713#event-15327680715 at 11/17/2024, 07:00 PM PST
- aswutmaxcy issue closed by PR 7713: https://github.com/hackforla/website/issues/7601#event-15327680818 at 11/17/2024, 07:00 PM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/6424#issuecomment-2573642511 at 01/06/2025, 10:08 AM PST
- aswutmaxcy closed as completed: https://github.com/hackforla/website/issues/6424#event-15829318922 at 01/06/2025, 10:08 AM PST
- aswutmaxcy assigned: https://github.com/hackforla/website/issues/7525 at 01/06/2025, 10:34 AM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/7525#issuecomment-2573689023 at 01/06/2025, 10:37 AM PST
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/7525#issuecomment-2602883136 at 01/20/2025, 08:48 AM PST
- aswutmaxcy unassigned: https://github.com/hackforla/website/issues/7525 at 03/17/2025, 06:39 PM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/8034#issuecomment-2837153019 at 04/28/2025, 06:01 PM PDT
- aswutmaxcy opened: https://github.com/hackforla/website/issues/8093 at 04/28/2025, 06:02 PM PDT
- aswutmaxcy assigned: https://github.com/hackforla/website/issues/7762 at 05/29/2025, 03:54 PM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/7762#issuecomment-2920765083 at 05/29/2025, 03:57 PM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/7762#issuecomment-2927933824 at 06/01/2025, 02:36 PM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/7762#issuecomment-2933081731 at 06/02/2025, 06:45 PM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/7762#issuecomment-2981696910 at 06/17/2025, 01:10 PM PDT
- aswutmaxcy opened: https://github.com/hackforla/website/pull/8193 at 06/18/2025, 02:21 PM PDT
- aswutmaxcy closed: https://github.com/hackforla/website/pull/8193#event-18218381701 at 06/18/2025, 02:22 PM PDT
- aswutmaxcy opened: https://github.com/hackforla/website/pull/8194 at 06/18/2025, 02:37 PM PDT
- aswutmaxcy closed: https://github.com/hackforla/website/pull/8194#event-18218597769 at 06/18/2025, 02:38 PM PDT
- aswutmaxcy opened: https://github.com/hackforla/website/pull/8195 at 06/18/2025, 02:41 PM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/issues/7762#issuecomment-2985779038 at 06/18/2025, 02:43 PM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/pull/8195#issuecomment-3005741668 at 06/25/2025, 11:25 AM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/pull/8195#issuecomment-3006139692 at 06/25/2025, 02:02 PM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/pull/8195#issuecomment-3025195472 at 07/01/2025, 12:01 PM PDT
- aswutmaxcy commented on: https://github.com/hackforla/website/pull/8195#issuecomment-3028475065 at 07/02/2025, 09:19 AM PDT
- aswutmaxcy PR was merged: https://github.com/hackforla/website/pull/8195#event-18435565718 at 07/02/2025, 09:21 AM PDT
- aswutmaxcy issue closed by PR 8195: https://github.com/hackforla/website/issues/7762#event-18435566205 at 07/02/2025, 09:21 AM PDT
- aswutmaxcy assigned: https://github.com/hackforla/website/issues/1398 at 09/25/2025, 03:40 PM PDT
- aswutmaxcy commented: https://github.com/hackforla/website/issues/1398#issuecomment-3336154405 at 09/25/2025, 03:43 PM PDT
- aswutmaxcy commented: https://github.com/hackforla/website/issues/1398#issuecomment-3344235856 at 09/28/2025, 01:42 PM PDT
- aswutmaxcy commented: https://github.com/hackforla/website/issues/6424#issuecomment-3358857261 at 10/01/2025, 08:08 PM PDT
- aswutmaxcy commented: https://github.com/hackforla/website/issues/1398#issuecomment-3378244203 at 10/07/2025, 11:46 AM PDT
- aswutmaxcy commented: https://github.com/hackforla/website/issues/1398#issuecomment-3487611899 at 11/04/2025, 11:04 AM PST
- aswutmaxcy unassigned: https://github.com/hackforla/website/issues/1398 at 11/21/2025, 01:15 PM PST
- aswutmaxcy issue was closed as completed: https://github.com/hackforla/website/issues/6424 at 03/01/2026, 10:45 AM PST

**Comment by HackforLABot** [comment]
@aswutmaxcy

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, August 19, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@aswutmaxcy

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, August 26, 2025 at 12:05 AM PST.</sub>

**Comment by aswutmaxcy** [comment]
1. Started a list of behaviors for checkboxes on the website, I'll start suggestions after
2. No blockers yet
3. Tomorrow and Friday I'll have more time to work on it
